src/stratis_cli/_error_reporting.py

```diff
@@ -151,6 +151,8 @@ def _interpret_errors(errors):
             )
             return fmt_str % error
 
+        if isinstance(error, BrokenPipeError):
+            return "Broken pipe"
```

Comment:
Please add a line or two between this and the comment below. There's a major separation between the code that checks the second error and the code that checks the last error, so there ought to be some visual separation. This method is too long, see the various pylint disables, but it is actually difficult to break it up into separate methods. So the best that can be done at this time is to use whitespace and comments.